cmd/context.go

```diff
@@ -103,6 +119,12 @@ func newContextCommand(config *settings.Config) *cobra.Command {
 
 	deleteContextCommand.Flags().BoolVarP(&force, "force", "f", false, "Delete the context without asking for confirmation.")
 
+	orgID = createContextCommand.Flags().String("org-id", "", "The id of your organization.")
```

Comment:
checked out the branch and tried running:
```bash
$ make build
$ ./build/darwin/amd64/circleci context create --help
Create a new context


Usage:
  circleci context create <vcs-type> <org-name> <context-name> [flags]

Flags:
  -h, --help   help for create

Global Flags:
      --host string         URL to your CircleCI host, also CIRCLECI_CLI_HOST (default "https://circleci.com/")
      --skip-update-check   Skip the check for updates check run before every command.
      --token string        your token for using CircleCI, also CIRCLECI_CLI_TOKEN
```

I think it'd be useful if the `--org-id` showed up under "Flags" 🙂